### PR TITLE
Add the news about blog of JournalStorage to `README.md`

### PR DESCRIPTION
**Pull request by eukaryo**
## Motivation

We posted a [new blog post](https://medium.com/optuna/introducing-the-stabilized-journalstorage-in-optuna-4-0-from-mechanism-to-use-case-e320795ffb61). I want to add the notification to `README.md` of this repository.

## Description of the changes

Updated `README.md`.

### CODE REVIEW

**Comment by eukaryo** [comment]
@c-bata @y0z Could you review this PR?

File: README.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ Optuna can dynamically construct the search spaces for the hyperparameters.
 
 ## :loudspeaker: News
 <!-- TODO: when you add a new line, please delete the oldest line -->
+* **Sep 18, 2024**: We posted [an article](https://github.com/optuna/optuna/releases) and check [our article](https://medium.com/optuna/introducing-the-stabilized-journalstorage-in-optuna-4-0-from-mechanism-to-use-case-e320795ffb61) about `JournalStorage`, which is stabilized in Optuna 4.0.
```
Comment (c-bata) [inline README.md:28]:
```suggestion
* **Sep 18, 2024**: We posted [an article](https://medium.com/optuna/introducing-the-stabilized-journalstorage-in-optuna-4-0-from-mechanism-to-use-case-e320795ffb61) about `JournalStorage`, which is stabilized in Optuna 4.0.
```

Comment (eukaryo) [reply README.md]: Sorry, and thank you for correcting my mistake.

**Review by c-bata — APPROVED** [review]
LGTM.

**Review by y0z — APPROVED** [review]
LGTM

**Pull request merged**